fedimint-client-legacy/src/lib.rs

```diff
@@ -727,20 +727,20 @@ impl<T: AsRef<ClientConfig> + Clone + MaybeSend> Client<T> {
         self.reissue(notes_to_reissue, rng).await
     }
 
-    pub async fn await_consensus_block_height(
+    pub async fn await_consensus_block_count(
         &self,
-        block_height: u64,
+        block_count: u64,
     ) -> std::result::Result<u64, task::Elapsed> {
         task::timeout(Duration::from_secs(30), async {
-            self.await_consensus_block_height_inner(block_height).await
+            self.await_consensus_block_count_inner(block_count).await
         })
         .await
     }
 
-    async fn await_consensus_block_height_inner(&self, block_height: u64) -> u64 {
+    async fn await_consensus_block_count_inner(&self, requested_count: u64) -> u64 {
         loop {
-            match self.context.api.fetch_consensus_block_height().await {
-                Ok(height) if height >= block_height => return height,
+            match self.context.api.fetch_consensus_block_count().await {
+                Ok(current_count) if requested_count <= current_count => return current_count,
```

Comment:
I hope there is not a off by one error here

Reply:
Double check me, but I've swapped current and requested as I like everything `a < b`, and they were renamed as they were poorly named.

Reply:
Perhaps it's better to keep the previous behavior?
```suggestion
                Ok(current_count) if requested_count < current_count => return current_count,
```

Reply:
Previous one was `>=` (now `<=` because I flipped them), no?

Reply:
Don't listen to me, I'm crazy

Reply:
Me too! :D